src/framesummary/FrameWatchConfigurationViewProvider.js

```diff
@@ -22,7 +22,11 @@ export default class FrameWatchConfigurationViewProvider {
 
   view(selection) {
     const domainObject = selection[0][0].context.item;
-    const tableConfiguration = new FrameWatchTableConfiguration(domainObject, openmct, this.type);
+    const tableConfiguration = new FrameWatchTableConfiguration(
+      domainObject,
+      this.openmct,
+      this.type
+    );
 
```

Comment:
I think we need the `const self = this;` then pass in `self.openmct` to the provide, like you've done elsewhere.